tests/robottelo/test_config_casts.py

```diff
@@ -60,6 +58,8 @@ def cast_logging_level(self):
         return casts.LoggingLevel()
 
     def test_cast_logging_level(self, cast_logging_level):
+        import logging
+
```

Comment:
should this be a log zero import? like `from robottelo.logging import logger`

Reply:
Not in this case, because we're using the log level constants defined in the built-in `logging` module. Good eye though!

Reply:
Okay gotcha, just a question, can't we just replace this with `logzero.loglevel` ? (optional)